packages/rum/src/index.d.ts

```diff
@@ -38,7 +38,10 @@ declare module '@elastic/apm-rum' {
      * undocumented, might be removed in future versions
      */
     isEnabled(): boolean
+    isActive(): boolean
+    fetchCentralConfig(): Promise<AgentConfigOptions | undefined>
```

Comment:
`fetchCentralConfig` is not meant to be used as an API. We should probably indicated that in the method comment but for now would you please remove this line.

Reply:
Done